modern/src/login/LoginPage.js

```diff
@@ -129,6 +131,12 @@ const LoginPage = () => {
     return () => handleLoginTokenListeners.delete(listener);
   }, []);
 
+  useEffect(() => {
+    if (oidcForced) {
+      document.location = '/api/session/openid/auth';
```

Comment:
Maybe extract this into a function so it can be reused here and in the button click callback?